modules/test/playwright/tests/osb-faro-web/segments.spec.ts

```diff
@@ -362,3 +368,264 @@ test('Search the Segment Profile Distribution', async ({apiHelpers, page}) => {
 		);
 	});
 });
+
+test('Segment Composition shows Active and Known individuals', async ({
+	apiHelpers,
+	page,
+}) => {
+	const anonymousIdentityID = '87';
+	const date1 = new Date();
+	const pageName = 'Liferay - AC Page';
+	const knownIndividualName = 'ac';
+	const dynamicSegmentName = 'Test Dynamic Segment';
+	const staticSegmentName = 'Test Static Segment';
+	const channelName = 'My Property - ' + getRandomString();
+
+	const {channel, project} = await createChannel({
+		apiHelpers,
+		channelName,
+	});
+
+	const generateIndividual = (name) => {
+		const id = getRandomString();
+
+		return {
+			id,
+			name,
+		};
+	};
+
+	const knownIndividual = [generateIndividual(knownIndividualName)];
+
+	await test.step('Create the known individuals directly in the AC database', async () => {
+		await createIndividuals({
+			apiHelpers,
+			individuals: knownIndividual,
+		});
+	});
+
+	await test.step('Create an identity for an anonymous directly in the AC database', async () => {
+		const identities = [
+			{
+				createDate: date1.toISOString(),
+				id: anonymousIdentityID,
+			},
+		];
+
+		await apiHelpers.jsonWebServicesOSBAsah.createIdentities(identities);
+	});
+
+	await test.step('Create events for the anonymous and known individual to appear in AC', async () => {
+		const knownIndividualEvent = knownIndividual.map((individual) => ({
```

Comment:
You probably can inline this on line 429 as well

Reply:
Done